run.py

```diff
@@ -28,24 +29,28 @@ def write_topics(repo, topiclist, dry_run):
 g = Github(token)
 
 try:
-    user_repositories = list(g.get_user().get_repos())
+    print("Getting list of repositories")
+    if args.organization:
+        repositories = [r for r in g.get_user().get_repos(affiliation = "organization_member", visibility = "public") if r.organization.login == args.organization]
```

Comment:
Won't this cause permission errors for organizations where I'm a member but don't have the admin rights to set topics?

Reply:
It propably does, but I'm not sure how to check for those permissions. There's https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-repository-permissions-for-a-user and also a permission dict in the repo response, but I couldn't find any information on the necessary permissions to set topics. We could just check for the admin permission, which is likely what it is...

Reply:
"Repository admins can add any topics they'd like to a repository." (https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/classifying-your-repository-with-topics#about-topics) - let's just check for the admin permission